example/associations/many-to-many-with-associations.js

```diff
@@ -0,0 +1,86 @@
+var Promise, collections, connections, setupWaterline;
+
+Promise = require('bluebird');
+
+setupWaterline = Promise.promisify(require('../raw/bootstrap'));
+
+connections = {
+    associations: {
+        adapter: 'sails-orientdb',
+        host: 'localhost',
+        port: 2424,
+        user: 'root',
+        password: 'root',
+        database: 'example-waterline-manyToMany',
+        options: {
+            databaseType: 'graph',
+            unsafeDrop: 'true',
```

Comment:
I don't think we need this, the less we put in the example code the easier it for people to understand what are the key elements.